gradle/dependency-versions.gradle

```diff
@@ -17,4 +17,5 @@ ext {
     commonsVersion = "1.5"
     bouncycastleVersion = "1.52"
     javaxVersion = "3.0.1"
+    helixVersion = "0.6.7-SNAPSHOT"
```

Comment:
this can't be merged in this state right? Do we have to wait until the relevant version of helix is published?